.github/workflows/common-flow-host.yml

```diff
@@ -0,0 +1,83 @@
+name: default (common) flow for host runner
+
+on:
+  workflow_call:
+    inputs:
+      env:
+        required: true
+        type: string
+      san:
+        type: string
+      get-redis:
+        type: boolean
+
+env:
+  MODE: "sudo"
+
+jobs:
+  common-flow:
+    runs-on: ${{ inputs.env }}
+# From this point on, this file should be identical to the other common flow.
+    defaults:
+      run:
+        shell: bash -l -eo pipefail {0}
+    steps:
+      - name: Deps checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - run: |
+          git config --global --add safe.directory '*'
+          git submodule update --init --recursive
+      - name: Setup
+        run: |
+          ./sbin/setup
+      # TODO: use this instead
+      # - name: Deps checkout
+      # uses: actions/checkout@v3
+      # with:
+      #     sparse-checkout: .install
+      # - name: Setup
+      #   run: |
+      #     ${{env.MODE}} ./.install/install_script.sh
+      # - name: Full checkout
+      #   uses: actions/checkout@v3
+      #   with:
+      #     submodules: recursive
+      #     set-safe-directory: "'*'"
+      - name: Get Redis
+        if: ${{ inputs.get-redis }}
+        run: |
+          export GITHUB_TOKEN="${{ secrets.GITHUB_TOKEN }}"
+          ./deps/readies/bin/getredis --with-github-token
+      - name: Build
+        run: |
+          make SAN=${{ inputs.san }}
+          make COORD=1 SAN=${{ inputs.san }}
+      - name: Unit tests
+        run: |
+          make unit-tests LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }}
+      - name: Flow tests
+        run: |
+          make pytest QUICK=1 LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }}
```

Comment:
always quick?

Reply:
by default (on every push) yes. Might change/parameterize this when we get to the full-required flow definition